code/game/turfs/open/lava.dm

```diff
@@ -1,4 +1,6 @@
-///LAVA
+/// LAVA
+/// PLASMA LAVA 
+/// MAFIA PLASMA LAVA
```

Comment:
These are triple slashes, so they're auto docs for the type paths. Put these above the individual datums...or just remove them, these are completely useless as documentation anyway.

Reply:
Applied with 7a3c6d3cc3fba426456997c04950abf0a88519a2